spec/router/dsl_spec.rb

```diff
@@ -264,4 +264,28 @@
       end
     end
   end
+
+  context "Mount" do
```

Comment:
Some tests are missing:

* for the hash version of `mount`;
* without the `via` option;
* with a class instead of a lambda;